StyleCop.Analyzers/StyleCop.Analyzers/Helpers/MemberOrderHelper.cs

```diff
@@ -158,6 +158,10 @@ internal static AccessLevel GetAccessLevelForOrdering(SyntaxNode member, SyntaxT
                     {
                         accessibility = AccessLevel.Internal;
                     }
+                    else if (member.Parent.IsKind(SyntaxKind.InterfaceDeclaration))
```

Comment:
Can't seem to add a comment higher up in this file, so adding it here instead.

How about the TypeMemberOrder property in this file and the code in lines 48 and 58? I don't recognize the behaviour that this code seems to imply, but that might be related to how I normally have StyleCop configured. Looks very much like records are missing here.

Reply:
Yes, this revealed another issue which is now fixed.

Reply:
Nothing else from me 👍